code/datums/components/squeak.dm

```diff
@@ -130,11 +130,11 @@
 		holder = null
 
 // Disposal pipes related shits
-/datum/component/squeak/proc/disposing_react(datum/source, obj/structure/disposalholder/holder, obj/machinery/disposal/source)
+/datum/component/squeak/proc/disposing_react(datum/source, obj/structure/disposalholder/disposal_holder, obj/machinery/disposal/disposal_source)
```

Comment:
BYOND doesn't error on duplicate args. Instead it just uses the first one. Renamed the other arg for consistency.